src/main/java/chatcat/tasks/Task.java

```diff
@@ -0,0 +1,63 @@
+package chatcat.tasks;
+
+import java.io.Serializable;
+
+/**
+ * The default Task class.
+ */
+public class Task implements Serializable {
+    private final String task;
+    private boolean isDone;
+
+    /**
+     * Creates a {@code Task} object using a specified description.
+     *
+     * @param task the description of this task.
+     */
+    public Task(String task) {
+        this.task = task;
+        this.isDone = false;
+    }
+
+    /**
+     * Marks a task {@code Task} as completed.
+     */
+    public void setDone() {
+        isDone = true;
+    }
+
+    /**
+     * Unmarks a task {@code Task} as not completed.
+     */
+    public void setUnDone() {
```

Comment:
```suggestion
    public void setUndone() {
```
Good naming of methods by using verbs here! Is it necessary to capitalize it in this manner though, since undone is a word itself.